### PR TITLE
Japanese translation work

### PR DESCRIPTION
**Pull request by ystk**
This pull request adds Japanese translation for the following files.
  introduction/01-introduction.asciidoc
  introduction/02-problem-solved.asciidoc
I have created a directory 'ja' under the introduction to store Japanese translations.
  introduction/ja/01-introduction-ja.asciidoc
  introduction/ja/02-problem-solved-ja.asciidoc
I have an issue to use a multibyte character from my command line. This is the reason why I use '-ja' prefix in the filename.

### CODE REVIEW

**Comment by lenucksi** [comment]
Hi @ystk thanks a lot for your translation! :tada: :tada:  
Unfortunately I can not read or write Japanese. I am more than happy to help merging this PR after it has been reviewed by a second person who speaks Japanese. Looking forward to see this progress - thanks again for your contribution! :smiley: 

As for the `-ja` prefix in the filename: This looks fine to me.

**Comment by johtani** [comment]
Hi, I'm new to InnerSource, but I can review Japanese translation. Can I help it?
I heard of this from @MaineC ;)

**Comment by lenucksi** [comment]
> Hi, I'm new to InnerSource, but I can review Japanese translation. Can I help it?
> I heard this from @MaineC ;)

Hey @johtani and thanks for reaching out and offering to help! Also, thanks to @MaineC for asking :smiley:
Your help would be very much welcome.

The thing to do here is to take a look at the corresponding English language content in the root folder and cross-read it against the proposed Japanese translation, e.g

introduction/01-introduction.asciidoc <-> introduction/ja/01-introduction-ja.asciidoc

What we're looking for is the following:
- Does the translated content reflect the English language content adequately.
- Are there any typos or similar small overlooked parts.

How to go about it:
- If there's anything to comment on, please use the review feature of GitHub to leave a review.
  - If you suggest changes, please try to use [GitHub's commit suggestions](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request). This way your contributions will be part of the Git commit history and you'll be properly recorded as a Contributor.
- If there's nothing and everything is fine, please also leave a review indicating the readiness to merge.
- Once I see that all open review discussions are resolved and there's an "Accepted" review on it, I'll gladly merge it.

Thanks again to both of you - @ystk and @johtani for doing the translation and offering to review it! :+1:

File: introduction/ja/03-how-works-ja.asciidoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+同様に、ほとんどのゲストは、その家やホストに対して敬意を表したいと感じています。
+They're careful with the items in the house and follow the rules for the duration of their stay.
+彼らは、滞在期間中、家の備品を丁寧に扱い、家のルールにも従います。
+彼らは、礼儀正しく丁寧に振る舞うことで、再度招かれることを期待しているかもしれません。
+こうした家を訪問する際の考え方は、コードベースに対するホストが、ゲストからのコントリビューションを受け入れる際に、チームが取るべき態度や行動の考え方とも言えます。
+
+それでは、InnerSourceのプロセスの仕組みがどのように機能するかを見てみましょう。
+この説明の補足として、ここでゲストチームとホストチームそれぞれのキーパーソンとなる人に名前を付けておきます。
+まず、https://innersourcecommons.org/resources/learningpath/product-owner/index[_プロダクトオーナー_] は、ホストチームが受け入れるコントリビューションがどの機能かを決定します。
+https://innersourcecommons.org/resources/learningpath/contributor/index[_コントリビューター_] は、ゲストチーム内の個人で、ホストチームにレビューしてもらうためにコードを投稿します。
```
Comment (johtani) [inline introduction/ja/03-how-works-ja.asciidoc:26]: There are spelling inconsistency in this file.「コントリビューター」「コントリビュータ」

File: introduction/ja/03-how-works-ja.asciidoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+https://innersourcecommons.org/resources/learningpath/trusted-committer/index[_トラステッドコミッター_] は、ホストチームを代表する人であり、コントリビュータがプルリクエストを正しく投稿するために必要な指導やタイムリーなサポートを提供します。
+小さな草の根活動的な取り組みでは、一人でプロダクトオーナーとトラステッドコミッター _両方_ の役目をすることがあります。
+
+これらの定義を用いて、InnerSourceのコントリビューションに関する基本的なアウトラインを説明すると、次のようになります。
+* ゲストチームやコントリビュータがホストチームからの機能を要求します。
+* プロダクトオーナーは、機能要求を表すユーザーストーリーがゲストチームまたはホストチームのメンバによって作られたものであるかを確認します。
+このユーザーストーリーでは、ゲストチームが同意できるように要求された機能を説明しなければなりません。
+例えば、こうした詳細にはアーキテクチャの制約、コーディング規約、依存関係の使用法、データコントラクトが含まれます。
```
Comment (johtani) [inline introduction/ja/03-how-works-ja.asciidoc:34]: Maybe missing one sentence. "They also list any details from the host team on how the feature should be delivered in order for the work to be accepted." Could you translate too?

File: introduction/ja/03-how-works-ja.asciidoc [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+* トラステッドコミッターのサポートで、コントリビュータはリクエストされた要求を実装するためのプルリクエストを送ります。
+こうした手順は、チームの時間や優先度の一般的な組織向けのもので、特定のシステムを仮定したものではないことに注意してください。InnerSourceは、チームがすでに組織化するための確立された方法を持っていることを想定ており、ホストにコードをコントリビューションしたいと思っているゲストチームがある場合に、連携して作業するためのフレームワークを提供しています。
```
Comment (johtani) [inline introduction/ja/03-how-works-ja.asciidoc:36]: 想定**し**ており

**Comment by johtani** [comment]
Hi @lenucksi , thanks for your guidance. 
I left some comments, e.g. typo correction and missing translations. 
I'll work with him until PR is fine, stay tuned :D

**Comment by rrrutledge** [comment]
Thanks, @johtani

File: introduction/ja/02-problems-solved-ja.asciidoc [hunk 4]
```diff
@@ -0,0 +1,37 @@
+== InnerSourceによって解決される課題とは？
+
+InnerSourceは、企業の組織構造やその立場に関係なく、誰もがコードを再利用したりコラボレーションすることを推奨し、それに報いることができるものです。
+このアプローチは、従来の組織に見られるアイデアや成果物を企業組織の階層やサイロの中に閉じ込めておくものとは異なるものです。
+この思想について例を示して、状況がどのようなものかを見ていきましょう。
```
[the remark this answers is not in the document]
Comment (ystk) [reply introduction/ja/02-problems-solved-ja.asciidoc:5]: 「この思想について例を示す状況がどのようなものかを見ていきましょう。」や「この考えの例を示す状況がどのようなものか見ていきましょう。」は、日本語で意味が取りにくいので、
「この考えについて実例を上げて見ていきましょう。」
と訳してはいかがでしょうか？

File: introduction/ja/02-problems-solved-ja.asciidoc [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+同じ会社にある二つのチームが、別々のソフトウェア部品を提供する時、片方のチームのソフトウェアが、もう一方のチームのソフトウェアに依存する状況を想像してください。
+もう少し具体的にユーザエクスペリエンスを例にすると、表示用データを取得するAPIに依存するサービスがあります。
+このような状況は、一つのチームが作成するソフトウェアが、数十人、数百人の利用される大企業では一般的なことです。
+
+利用側のチームが多くの機能を必要とした時、提供側のチームは通常、どの機能から開発を進めるかを決めるために、ある種の要件や優先度付けを行うプロセスを持っています。
+すぐに作業に取り掛かることが必要な重要な必須機能に関するリクエストについては、利用側のチームは通常、次に示す3つのオプションから一つを選択することになると思いますが、それぞれ欠点があります。
```
[the remark this answers is not in the document]
Comment (ystk) [reply introduction/ja/02-problems-solved-ja.asciidoc:12]: 「すぐに作業に取り掛かるため優先度が付けられていなかった重要機能のリクエストのために、利用側のチームは通常、次に示す3つのオプションから一つを選択することになると思いますが、それぞれ欠点があります。」としてみてはいかがでしょう？

**Comment by ystk** [comment]
Thank you very much for the review. I edited my local copy and pushed it to GitHub again.

File: introduction/ja/03-how-works-ja.asciidoc (hunk 2 above)
Comment (ystk) [reply introduction/ja/03-how-works-ja.asciidoc:35]: I translated the sentence. Yoshi can find on my latest commit.

**Comment by ystk** [comment]
I think there are three suggestions that need to be discussed.

File: introduction/ja/02-problems-solved-ja.asciidoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (johtani) [reply introduction/ja/02-problems-solved-ja.asciidoc]: とてもいいと思います。

File: introduction/ja/02-problems-solved-ja.asciidoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (johtani) [reply introduction/ja/02-problems-solved-ja.asciidoc]: こちらもとてもいいと思います。これで行きましょう。

File: introduction/ja/03-how-works-ja.asciidoc (hunk 2 above)
Comment (johtani) [reply introduction/ja/03-how-works-ja.asciidoc:35]: ありがとうございます、問題ないです。

File: introduction/ja/02-problems-solved-ja.asciidoc [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+ .. 会社全体として、同じ課題に対する重複したプロジェクトとコードを取得していしまいました。
+. *エスカレート：* 利用側のチームは、「No」という答えを受け入れずに、代わりに提供側のマネージメント層に影響（や強制）を与えるように働きかけます。
+このオプションは、利用側のチームにとっては、彼らが何も開発したりメンテナンスしたりせずに要求する機能を手に入れることができるため、魅力的に思えます。
+しかし、それは利用側のチームにとって結局足を引っ張ることになります。なぜなら、エスカレーションという開発に関係のない作業に注力しなければならないからです。
+加えて、このオプションは、何度も使えるものでもなくスケールしません。度重なるエスカレーションは、利用する側の信頼を損なうことに繋がるからです。
+エスカレーションは、提供側のチームにも同様（もしくはそれ以上）に混乱をもたらします。なぜなら、エスカレーションされた機能の処理を、通常のワークフローや優先度付けの方法の範囲外で行わなければならないからです。
+
+この議論は、InnerSourceのための準備となります。
+InnerSourceは、利用側のチームが機能要求を通して必要なものが得られない、似たような状況に適用されます。
+InnerSourceは、 _静観_ 、 _回避策_ 、エスカレートの効果を関連する欠点なしに得るための方法を提供します。
```
Comment (johtani) [inline introduction/ja/02-problems-solved-ja.asciidoc:33]: _エスカレート_

**Comment by rrrutledge** [comment]
Thanks for this work!

File: introduction/ja/02-problems-solved-ja.asciidoc (hunk 6 above)
Comment (ystk) [reply introduction/ja/02-problems-solved-ja.asciidoc]: Thank you. I fixed it.

**Comment by lenucksi** [comment]
> So, LGTM all changes.

Excellent. Thanks for the review @johtani and addressing the review @ystk comments! :smiley: :tada: :+1: 
Could you please resolve (i.e. click the "resolve conversation" buttton) the open conversations if they are resolved (which they probably are if everything LGTM for you.)?

**Pull request merged**